pep-0676.rst

```diff
@@ -9,20 +9,22 @@ Content-Type: text/x-rst
 Created: 01-Nov-2021
 Post-History: 23-Sep-2021
 
+
 Abstract
 ========
 
 This PEP addresses the infrastructure around rendering PEP files from
-reStructuredText_ files to HTML webpages. We aim to specify a self-contained
+`reStructuredText`_ files to HTML web pages. We aim to specify a self-contained
 and maintainable solution for PEP readers, authors, and editors.
 
+
 Motivation
 ==========
 
 As of November 2021, Python Enhancement Proposals (PEPs) are rendered in a
-multi-system, multi-stage process. A continuous integration (CI) task runs a
-docutils_ script to render all PEP files individually. The CI task then uploads
-a tar archive to a server, where it is retrieved and rendered into the
+multi-system, multi-stage process. A Continuous Integration (CI) task runs a
+`docutils`_ script to render all PEP files individually. The CI task then
+uploads a TAR to a server, where it is retrieved and rendered into the
```

Comment:
I'd also keep this as "tar archive".

The word "tar" is commonly written in lowercase: https://en.wikipedia.org/wiki/Tar_(computing)

(It's originally from "<b>t</b>ape <b>ar</b>chive, so not strictly an initialism.)

So archive in "tar archive" is kind of redundant, but it's very common to write "tar archive" and I feel it's clearer, and, well, we're not dealing with tapes.

```suggestion
uploads a tar archive to a server, where it is retrieved and rendered into the
```

Reply:
You're correct on the lowercase, of course (its an extension and a CLI program, not an acronym, and at least as I've always pronounced it, its an acronym, not an initialism), and as much as the redundancy irks me, I can see the clarity benefit (for which reason I almost backed out the change originally), and I'm firmly in the camp that clarity trumps aesthetics, purity and other considerations if they are in conflict and cannot be resolved through other means.

Reply:
Pedantic side note @hugovk — have I been pronouncing tar (the archive) wrong all this time? As an acronym in the strict sense of pronunciation ("tar", one word) rather than an initialism ("t-a-r", single letters)?

On a related side-side note, when I was younger I used to insist on pronouncing my name as an initialism ("C-A-M") rather than an acronym ("Cam"), since they _were_ technically my first three initials after all, but everyone called me "CAM" anyway, so I eventually just embraced it.

Reply:
I have only heard and say "tar" as one word, but who knows!

"CAM" instead of "C-A-M": people say "NASA" rather than "N-A-S-A" :)

Reply:
> I have only heard and say "tar" as one word, but who knows!

Yeah, that's all I've heard too haha; my (pedantic) comment was based on your reference to it as an initialism rather than an acronym, heh.

> "CAM" instead of "C-A-M": people say "NASA" rather than "N-A-S-A" :)

And I should know, since I work there :)

Reply:
@CAM-Gerlach  -- I've been addressing you as Chris / Christopher per your logo-name thing -- would you prefer Cam?
A

Reply:
Yeah, sure :) While "CAM" is in the center of my logo-name thing, you're not the first person to look at it closely and see the "Christopher" instead, heh